jarviscli/plugins/spinthewheel.py

```diff
@@ -0,0 +1,20 @@
+from plugin import plugin
+import random
+'''This code picks one of the random inputs given by the user
+like spin wheel'''
+
+
+@plugin("spin wheel")
+def spin(jarvis, s):
+    jarvis.say(' ')
+    jarvis.say('welcome to spin the wheel\n')
+    jarvis.say('enter the number of elements in the wheel')
+    num = jarvis.input()
+    num = int(num)
```

Comment:
Crashes if you don't enter a number

```

~> What can I do for you?
spin wheel
 
welcome to spin the wheel

enter the number of elements in the wheel
notanumber
Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "/home/phil/git/Jarvis/jarviscli/CmdInterpreter.py", line 223, in try_do
    do(self, s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 217, in run
    self.get_plugins(command).run(jarvis, new_s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 209, in run
    self._backend[0](jarvis.get_api(), s)
  File "/home/phil/git/Jarvis/jarviscli/plugins/spinthewheel.py", line 13, in spin
    num = int(num)
ValueError: invalid literal for int() with base 10: 'notanumber'
```

Either check this error or (better) use jarvis.input_number
https://github.com/sukeesh/Jarvis/blob/master/doc/API.md#input_number

Reply:
Thanks and I will look into it